pkg/network/cache/types.go

```diff
@@ -9,10 +9,19 @@ import (
 	v1 "kubevirt.io/client-go/api/v1"
 )
 
+type PodIfaceState int
+
+const (
+	PodIfaceNetworkPreparationPending PodIfaceState = iota
```

Comment:
I find this state redundant. There is no code using it.
But maybe it is more readable, don't know...

Reply:
We need a way to denote the initial state wich is this.